modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/context/index.js

```diff
@@ -96,9 +96,14 @@ const AppContextProvider = ({assetsPath, children, page}) => {
 						({name}) => name === ROLE_TYPES.admin.key
 					);
 
+				const isStaff = data.userAccount?.roleBriefs?.some(
+					(role) => role.name === 'Administrator'
```

Comment:
Import & use the ROLE_TYPES variable here 
```suggestion
					(role) => role.name === ROLE_TYPES.admin.name
```